tempest/api/workloadmgr/multiattach_volumes/test_multiattach_volumes.py

```diff
@@ -333,8 +425,7 @@ def test_01_multiattach_volumes(self):
                     self.wid, self.snapshot_id2)
                 LOG.debug(f"snapshot_found: {self.snapshot_found}")
                 if self.snapshot_found:
-                    reporting.add_test_step("Verify snapshot existence on " \
-                                            "target backend", tvaultconf.PASS)
+                    reporting.add_test_step("Verify snapshot existence on target backend", tvaultconf.PASS)
```

Comment:
get_mountpoint_path() and check_snapshot_exist_on_backend() calls should be updated as per 5.0-beta code.